src/calculateRentalCost.js

```diff
@@ -1,10 +1,12 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Avoid using literals in expressions. Define constants for the threshold number of days and the corresponding discounts to make the code more readable and maintainable.